src/reducers/index.js

```diff
@@ -89,15 +89,14 @@ function reducers(state = DEFAULT_STATE, action) {
       const { props } = action;
 
       let newState = setStateFromProps(state, props);
-      newState = columnStateHelper.initialize(newState, props, {});
       newState = initializeRowHeights(newState);
       const scrollAnchor = getScrollAnchor(newState, props);
-      return computeRenderedRows(newState, scrollAnchor);
+      newState = computeRenderedRows(newState, scrollAnchor);
+      return columnStateHelper.initialize(newState, props, {});
```

Comment:
Since column state work depends on knowing if the vertical scrollbar is visible, and that depends on knowing the content height, I had to move this after the computeRenderedRows calls